resources/lrsql/config/test/default/webserver.edn

```diff
@@ -9,6 +9,8 @@
  :ssl-port                  8443
  :http-host                 "0.0.0.0"
  :http-port                 8080
+ :allow-all-origins         false
+ :allowed-origins           nil
```

Comment:
I wonder if you can actually set `allowed-origins` to some array and then test the CORS setting using a `babshka.curl` request.

Reply:
see https://github.com/yetanalytics/lrsql/pull/265/commits/06d2bfc5116eb0f8b6e928a993bcd4f7bb02939e